.github/workflows/ci-deploy.yml

```diff
@@ -77,78 +79,29 @@ jobs:
+    - name: Kick off Terraform deploy in sysops/
       id: sysops-deploy
+      if: ${{ github.event.pull_request.merged }}
       run: |
-        BRANCH=$(echo "${GITHUB_REF#refs/*/}" | sed 's/.*\///')
-        if [[ "$BRANCH" != master ]]; then
-          curl \
+        curl \
           -X POST \
           -H "Accept: application/vnd.github+json" \
           -H "Authorization: Bearer ${{ secrets.SYSOPS_RW_GITHUB_TOKEN }}" \
           -H "X-GitHub-Api-Version: 2022-11-28" \
           https://api.github.com/repos/meedan/sysops/actions/workflows/deploy_${{ github.event.repository.name }}.yml/dispatches \
-          -d '{"ref": "master", "inputs": {"git_sha": "${{ github.sha }}", "branch": "'$BRANCH'"}}'
-        fi
-
-    - name: Send GitHub Action trigger data to Slack workflow on success
-      id: slack-api-notify-success
-      if: ${{ success() }}
-      uses: slackapi/slack-github-action@v1.23.0
-      with:
-         payload: |
-          {
-            "attachments": [
-              {
-                "color": "#00FF00",
-                "blocks": [
-                  {
-                    "type": "section",
-                    "text": {
-                      "type": "mrkdwn",
-                      "text": "Kicked off by: ${{ github.triggering_actor }}\nWorkflow: https://github.com/meedan/presto/actions/runs/${{ github.run_id }}"
-                    }
-                  },
-                  {
-                    "type": "section",
-                    "text": {
-                      "type": "mrkdwn",
-                      "text": "Presto Deploy:\n${{ github.event.pull_request.html_url || github.event.head_commit.url }}"
-                    }
-                  }
-                ]
-              }
-            ]
-          }
-      env:
-        SLACK_WEBHOOK_URL: ${{ secrets.CHECK_DEV_BOTS_SLACK_WEBHOOK_URL }}
-        SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
+          -d '{"ref": "master", "inputs": {"git_sha": "${{ github.sha }}"}}'
 
-    - name: Send GitHub Action trigger data to Slack workflow on failure
-      id: slack-api-notify-failure
-      if: ${{ failure() }}
-      uses: slackapi/slack-github-action@v1.23.0
-      with:
-         payload: |
-          {
-            "attachments": [
-              {
-                "color": "#FF0000",
-                "blocks": [
-                  {
-                    "type": "section",
-                    "text": {
-                      "type": "mrkdwn",
-                      "text": "Presto Deploy failed\nWorkflow: https://github.com/meedan/presto/actions/runs/${{ github.run_id }}"
-                    }
-                  }
-                ]
-              }
-            ]
-          }
-      env:
-        SLACK_WEBHOOK_URL: ${{ secrets.ITS_BOTS_SLACK_WEBHOOK_URL }}
-        SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
+    - name: Kick off Terraform deploy in sysops/
+      id: sysops-deploy
+      if: github.event_name == 'create' && startsWith(github.ref, 'refs/tags/v')
```

Comment:
you might want to double check the difference between create events and push events, this could work with specific process controls but they are separate events